server/helpers/youtube-dl/youtube-dl-base-builder.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  CONSTRAINTS_FIELDS,
+  VIDEO_CATEGORIES,
+  VIDEO_IMPORT_LIVE_STATUSES,
+  VIDEO_LANGUAGES,
+  VIDEO_LICENCES
+} from '@server/initializers/constants'
+import { peertubeTruncate } from '@server/helpers/core-utils'
+import { isUrlValid } from '@server/helpers/custom-validators/activitypub/misc'
+
+class YoutubeDLBaseBuilder {
+  public normalizeObject (obj: any) {
```

Comment:
Please remove the `public` keyword for consistency

Reply:
Done